katas/content/single_qubit_gates/y_gate/solution.md

```diff
@@ -1 +1,10 @@
-## Solution
+### Solution
+
+Use the `Y` operation from the `Microsoft.Quantum.Instrinsic` namespace.
```

Comment:
Why are the solutions markdown files and not code files? I would expect we'd have a 'Show solution' type button in the editor for the exercise which would populate with the solution code. (Which could have explanatory comments like line 3 here if needed)